src/plugins/lightscript.js

```diff
@@ -143,7 +202,7 @@ pp.parseWhiteBlock = function (allowDirectives?, isExpression?) {
   // must start with colon or arrow
   if (isExpression) {
     this.expect(tt.colon);
-    if (!this.isLineBreak()) return this.parseMaybeAssign();
+    if (!this.isLineBreak()) return this.parseWhiteBlockBody(node, indentLevel, true, false);
```

Comment:
Right? Would this obviate the need for `couldBeginStatement` and `parseWhiteBlockBody`?